LICENCE

```diff
@@ -1,6 +1,4 @@
-Copyright (c) 2005-2022 Citrix Systems Inc.
-Copyright (c) 2022-12-01 Cloud Software Group Holdings, Inc.
-All rights reserved.
+Copyright (c) 2005-2023 Cloud Software Group, Inc.
```

Comment:
Isn't it better to keep "Copyright (c) 2005-2022 Citrix Systems Inc."? Did you make this change by any official guides?

Reply:
I followed up https://info.citrite.net/display/xenserver/CSG+XenServer+Rebrand+Requirements to make the change. I will double check whether we need keep "Copyright (c) 2005-2022 Citrix Systems Inc."